DatabaseAndReporting/Integration/02_ManageQueryCategories.md

```diff
@@ -0,0 +1,218 @@
+[!!User interface Query categories](../UserInterface/03b_QueryCategories.md)
+[!!Manage the queries](./01_ManageQueries.md)
+[!!Manage the user rights](./05_ManageUserRights.md)
+
+# Manage the query categories
+
+Query categories serve the logical organization of the managed queries. The managed queries are then organized in a tree structure of folders and sub-folders.
+
+Query categories can be created, edited, deleted, and divided into sub-categories. All individual managed queries can, in turn, be assigned to a category and/or a sub-category, see [Create a query](./01_ManageQueries.md#create-a-query). Query category read and save rights can also be restricted or granted based on the user group, see [Grant access rights to a user group](./05_ManageUserRights.md#grant-access-rights-to-a-user-group).
+
+
+## Create a query category
+
+Create a query category for new managed queries you want to assign to a certain category.
+
+#### Prerequisites 
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Database and reporting > Settings > Tab QUERY CATEGORIES*
+
+![Query categories](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategories.png "[Query categories]")
+
```

Comment:
Grundsätzlich: Ist das ein Fehler im UI oder ein Bug, dass man nicht mehr zurückkommt, weil man sofort in der *Edit tree* view ist und sowohl ein Back als auch ein Cancel Button angezeigt werden, die nichts machen?